src/main/asciidoc/_chapters/hbase_mob.adoc

```diff
@@ -181,84 +323,59 @@ suit your environment, and restart or rolling restart the RegionServer.
 ----
 ====
 
-=== MOB Optimization Tasks
-
 ==== Manually Compacting MOB Files
 
 To manually compact MOB files, rather than waiting for the
-<<mob.cache.configure,configuration>> to trigger compaction, use the
-`compact` or `major_compact` HBase shell commands. These commands
+periodic chore to trigger compaction, use the
+`major_compact` HBase shell commands. These commands
 require the first argument to be the table name, and take a column
-family as the second argument. and take a compaction type as the third argument.
+family as the second argument. If used with a column family that includes MOB data, then
+these operator requests will result in the MOB data being compacted.
 
 ----
-hbase> compact 't1', 'c1’, ‘MOB’
-hbase> major_compact 't1', 'c1’, ‘MOB’
+hbase> major_compact 't1'
+hbase> major_compact 't2', 'c1’
```

Comment:
Would scripts that have the 'MOB' parameter still work? Just from this doc is not clear.

Reply:
Let me confirm if it's ignored or if it errors.

Reply:
All the shell, java, and rpc stuff is still in place for talking about 'MOB' type compactions is still present. so it'll work just do something different.

I'll try to make that clear in the upgrade consideration section.

Reply:
added a call out with example in the upgrade section